src/engine/BMGame.php

```diff
@@ -1882,13 +1882,13 @@ public function set_turbo_sizes(array $turboSizeArray) {
             $die = $this->playerArray[$playerIdx]->activeDieArray[$dieIdx];
 
             if (!$die->has_skill('Turbo')) {
-                $this->message = 'Cannot set turbo size for a non-turbo die';
-                return FALSE;
+                // ignore any turbo values supplied for non-turbo dice
```

Comment:
So, I was thinking about this, and I guess it would also be possible to change the implementation of the form submission to submit only values from non-disabled selectors. That would mean that I could revert both of the backend changes here.